src/lib.rs

```diff
@@ -964,6 +1203,7 @@ impl<'a> Drop for SwapChainOutput<'a> {
 }
 
 impl SwapChain {
+    /// Returns the next texture to be presented by the swapchain for drawing.
```

Comment:
Let's mention that the presentation happens upon dropping `SwapChainOutput` object.

Reply:
see https://github.com/gfx-rs/wgpu-rs/pull/11/commits/96e8784098076e6491cb8179e6fd01f51a62574f#diff-b4aea3e418ccdb71239b96952d9cddb6R1213